semantic_conventions/trace/messaging.yaml

```diff
@@ -129,6 +108,20 @@ groups:
         brief: >
           A unique identifier for the client that consumes or produces a message.
         examples: ['client-5', 'myhost@8742@s8083jm']
+      - ref: messaging.destination.name
+        requirement_level:
+          conditionally_required: If span describes operation on a single message or if the value applies to all messages in the batch.
+      - ref: messaging.destination.template
+        requirement_level:
+          conditionally_required: >
+            If available. Instrumentations MUST NOT use `messaging.destination.name` as template
+            unless low-cardinality of destination name is guaranteed.
```

Comment:
I think the second sentence should be moved somewhere else, it is not part of the requirement. Probably it should be part of the `note:` in the base referenced primary definition already.

Reply:
It's somehow already mentioned in the attribute `note`:

```
Destination names could be constructed from templates.
An example would be a destination name involving a user name or product id.
Although the destination name in this case is of high cardinality,
the underlying template is of low cardinality and can be effectively
used for grouping and aggregation.
```
I think the whole sentence "once more" here doesn't hurt and good overall for the "conditionally_required" section. People might miss reading or or the other.